Tasks/string_builder.py

```diff
@@ -0,0 +1,52 @@
+class StringBuilder:
+
+    def __init__(self, data):
+        self.string = list(data)
+
+    def append(self, elem: str):
+        self.string.extend(list(elem))
+
+    def __str__(self):
+        return ''.join(self.string)
+
+    def insert(self, elem: str, ind: int = 0):
+        if ind > len(''.join(self.string)):
+            raise IndexError('Index out of range')
+        if ind < 0:
+            self.string.insert(ind, elem)
+        else:
+            self.string.insert(ind, elem)
+
+    def __getitem__(self, key: int or slice):
```

Comment:
```suggestion
    def __getitem__(self, key: Union[int, slice]):
```
also need to import Union from typing